commands.go

```diff
@@ -61,7 +61,7 @@ type statefulCmdable struct {
 
 //------------------------------------------------------------------------------
 
-func (c statefulCmdable) Auth(password string) *StatusCmd {
+func (c *statefulCmdable) Auth(password string) *StatusCmd {
```

Comment:
While it may indeed be quicker, the difference will be in the pico-seconds range. I think we should follow https://golang.org/doc/faq#methods_on_values_or_pointers:

> Next is consistency. If some of the methods of the type must have pointer receivers, the rest should too, so the method set is consistent regardless of how the type is used. See the section on method sets for details.